mypy.ini

```diff
@@ -10,6 +10,7 @@ mypy_path=src
 
 # Useful settings.
 show_column_numbers=True
+show_error_codes=True
```

Comment:
Can you please add a comment why turning this option on is useful?

Reply:
Should we then explain the other options in this file as well?

Reply:
Maybe if they are interesting like this one :wink:

Reply:
Commit https://github.com/jenstroeger/python-package-template/pull/109/commits/4a3b0fbf8df2ad84c9d4fdb82e1a15d4b2b9a0d4.